openpype/tools/creator/widgets.py

```diff
@@ -99,7 +99,7 @@ def __init__(self):
 
     def validate(self, text, pos):
         results = super(SubsetNameValidator, self).validate(text, pos)
-        if results[0] == self.Invalid:
```

Comment:
@iLLiCiTiT can you confirm these changes are ok?

Reply:
Yes, ok.